pkg/capture/capture.go

```diff
@@ -272,6 +272,9 @@ func (c *Capture) capturePacket() error {
 	c.errCount++
 	c.errMap[err.Error()]++
 
+	// add error counter to exposed metric
+	captureErrors.Inc()
```

Comment:
I'm wondering: Can't we move this to the 5min interval as well (because we're already counting the value 4 lines up, so instead of duplicating this operation we could simply transfer the value together with the number of processed / dropped values further below)... IMHO it doesn't make sense to count the same number twice...

Reply:
Makes a lot of sense, yeah.

Reply:
Afaik that's why we have the error map (because that is persisted until writeout, no? Logging in the process routine (when a singular error occurs) is of course out of the question, but I always thought that the error map was intended exactly to solve that conundrum (and also to not spam the reader of the logs). But maybe that was exactly how it was done in the old API, I don't remember TBH... But yeah, let's tackle it in the linked extra issue (although I think that "when it's about understanding what kind of errors we even observe" is now, the metrics will not cover that)...

Reply:
I think it's even simpler than that, check out my comment here: https://github.com/els0r/goProbe/issues/178#issuecomment-1670988198